File: Python/Slice/is_slice_empty.py
# Check if a given slice is empty.
def is_slice_empty(provided_slice):
    return len(provided_slice) == 0

def main():
    randomSlice = ["c", "a", "b"]
    # Check if the slice is empty
    print(is_slice_empty(randomSlice))
    
main()

File: Rust/src/Slice/is_slice_empty.rs
fn main() {
    // Create a slice of random elements.
    let random_list_of_elements = ["one", "two", "three", "four", "five"];
    // Check if the slice is empty.
    println!("{}", is_slice_empty(&random_list_of_elements));
}


// Check if a given slice is empty.
fn is_slice_empty(provided_slice: &[&str]) -> bool {
    return provided_slice.is_empty();
}
